.github/workflows/test.yml

```diff
@@ -61,6 +61,69 @@ jobs:
 
       - name: Run just hypothesis tests with more examples
         run: pytest -n 4 tests --disable-warnings -m hypothesis --generate-plots
+          
+  rms:
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {
+              name: "RMS 12.1.4, 13.0.3, 13.1.0, 14.1.1",
+              os: ubuntu-20.04,
+              python: 3.8.6,
+              pip: 23.2.1,
+              wheel: 0.37.1,
+              setuptools: 63.4.3,
+              matplotlib: 3.3.2,
+              numpy: 1.19.2,
+              pandas: 1.1.3,
+              scipy: 1.5.3,
+            }
+          - {
+              name: "RMS 14.2",
+              os: ubuntu-latest,
+              python: 3.11.3,
```

Comment:
Interestingly GitHub Actions doesn't keep this version cached on runners but can still download and set it up from scratch